leeseulah/likelion-crud/src/main/java/com/likelion/likelioncrud/tag/api/dto/response/TagInfoResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com.likelion.likelioncrud.tag.api.dto.response;
+
+import com.likelion.likelioncrud.tag.domain.Tag;
+
+public record TagInfoResponseDto(
+        Long id,
+        String name
+) {
+    public static TagInfoResponseDto from(Tag tag) {
+        return new TagInfoResponseDto(tag.getId(), tag.getName());
```

Comment:
여기서는 저희가 계속 붙여줬던 `@Builder`를 사용해 빌더 패턴으로 작성해 줄 수 있겠죠 ?!

```suggestion
        return TagInfoResponseDto.builder()
                .id(tag.getId())
                .name(tag.getName())
                .build();
    }
```

Reply:
감사합니당 ,,
`public static TagInfoResponseDto from(Tag tag) {
        return TagInfoResponseDto.builder()
                .name(tag.getName())
                .build();
    }`